src/ray/protobuf/serve.proto

```diff
@@ -242,3 +242,60 @@ message PredictResponse {
 service PredictAPIsService {
   rpc Predict(PredictRequest) returns (PredictResponse);
 }
+
+// Used for gRPC proxy health check
+message RoutesRequest {}
+
+message RoutesResponse {
+  repeated string application_names = 1;
+}
+
+message HealthzRequest {}
+
+message HealthzResponse {
+  string response = 1;
```

Comment:
what is the response here? if it's a status message, let's call it `message` or `status`

Reply:
It will just be returning text "success", will change to message.